tests/Analytics/AnalyticsTest.php

```diff
@@ -30,15 +29,15 @@ class AnalyticsTest extends TestCase
 
     public function setUp(): void
     {
-        $this->ga = new GoogleAnalytics(App::getEnv('GA_TID'), App::getEnv('GA_CID'));
-        $this->pa = new Plausible(App::getEnv('PA_DOMAIN'), App::getEnv('PA_APIKEY'), 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36', '192.168.0.1');
-        $this->orbit = new Orbit(App::getEnv('OR_WORKSPACEID'), App::getEnv('OR_APIKEY'), 'Utopia Testing Suite');
-        $this->mp = new Mixpanel(App::getEnv('MP_PROJECT_TOKEN'));
-        $this->hs = new HubSpot(App::getEnv('HS_APIKEY'));
+        $this->ga = new GoogleAnalytics(getenv('GA_TID') ?? '', getenv('GA_CID') ?? '');
```

Comment:
Why getenv instead of App::getEnv?

Reply:
Removes our dependency on Utopia-PHP/Framework, plus us switching from `App` to `Http` broke it,

Reply:
Http should also have it iirc